docs/services/hpc-console.md

```diff
@@ -0,0 +1,53 @@
+[](){#hpc-console}
+# HPC Console
+
+HCP Console is a web application designed to provide essential functionalities for interacting with [FirecREST API v2 backend][ref-firecrest].
+It provides users with a modern web-based environment to interact with HPC resources such as clusters, job schedulers, and filesystems.
+
+## Features
+
+- A modern web interface built on FirecREST API functionalities.
+- A dashboard offering an overview of configured clusters and their statuses.
+- A dedicated view for creating and monitoring jobs.
+- A file navigator with basic file management functionalities, including upload and download features.
+
+A description of the views and functionalities can be found [here](https://eth-cscs.github.io/firecrest-ui/documentation/).
+
+## Views overview
+
+- Dashboard – shows cluster availability and basic system status.
+- Jobs – allows creating, submitting, and monitoring jobs. Users can provide job name, script, and resource requirements.
+- File Navigator – supports browsing directories, uploading, downloading, and deleting files. Editing files inline is not supported.
+
+## Supported version
+
+Starting early 2025, CSCS has introduced a new version of the API: [FirecREST version 2](https://eth-cscs.github.io/firecrest-v2).
+
+This web interface is available and runs on the FirecREST API v2 backend, FirecREST API v1 is not supported.
+
+## Alps platforms
+
+HPC console is currently available on three [Alps platforms][ref-alps-platforms].
+
+| Platform       | URL               | Clusters                                                                 |
+|----------------|----------------------|--------------------------------------------------------------------------|
+| HPC Platform   | [https://my.hpcp.cscs.ch/](https://my.hpcp.cscs.ch/) | [Daint](../clusters/daint), [Eiger](../clusters/eiger)             |
+| ML Platform    | [https://my.mlp.cscs.ch/](https://my.mlp.cscs.ch/)  | [Bristen](../clusters/bristen), [Clariden](../clusters/clariden)  |
+| CW Platform    | [https://my.cwp.cscs.ch/](https://my.cwp.cscs.ch/)  | [Santis](../clusters/santis)  |
```

Comment:
```suggestion
| HPC Platform   | [https://my.hpcp.cscs.ch/](https://my.hpcp.cscs.ch/) | [Daint][ref-cluster-daint], [Eiger][ref-cluster-eiger]             |
| ML Platform    | [https://my.mlp.cscs.ch/](https://my.mlp.cscs.ch/)  | [Bristen][ref-clsuter-bristen], [Clariden][ref-cluster-clariden]  |
| CW Platform    | [https://my.cwp.cscs.ch/](https://my.cwp.cscs.ch/)  | [Santis][ref-cluster-santis]  |
```

Additionally, it may be useful to add these links to each of the cluster pages (instead of? or additionally? not quite sure). They already mention jupyterhub and firecrest as "services" available. See e.g. `docs/clusters/eiger.md`.